src/Tests/ArgumentForwarding.Tests/ArgumentForwarding.Tests.csproj

```diff
@@ -11,7 +11,7 @@
     <ProjectReference Include="..\Microsoft.NET.TestFramework\Microsoft.NET.TestFramework.csproj" />
   </ItemGroup>
 
-  <Target Name="PrecompileScriptBuild" BeforeTargets="Build" Condition=" '$(IsCrossTargetingBuild)' != 'true' ">
+  <Target Name="PrecompileScriptBuild" BeforeTargets="Build" Condition=" '$(IsCrossTargetingBuild)' != 'true' and '$(DotNetBuildFromSource)' != 'true'">
```

Comment:
📝 Test projects should automatically be excluded from source build. If this project is not being detected as a test project (e.g. because it ends with Tests.csproj instead of UnitTests.csproj), you can manually set `<ExcludeFromSourceBuild>` to true. Once the test project is excluded from source build, it should not be necessary to modify the targets it contains.